pkg/operator/ceph/object/objectstore.go

```diff
@@ -195,6 +195,17 @@ func deleteSingleSiteRealmAndPools(objContext *Context, spec cephv1.ObjectStoreS
 // This is used for quickly getting the name of the realm, zone group, and zone for an object-store to pass into a Context
 func getMultisiteForObjectStore(clusterdContext *clusterd.Context, spec *cephv1.ObjectStoreSpec, namespace, name string) (string, string, string, error) {
 	ctx := context.TODO()
+
+	if spec.IsExternal() {
+		/* *
+		 * Currently external cluster with zones/zonegroup/realm are not supported, will be fixed by https://github.com/rook/rook/issues/6342
+		 * So if user does not create zone/zonegroup, RGW internally creates zone/zonegroup named as `default`, hence Rook can set this value in
+		 * the object context fields.
+		 */
```

Comment:
Currently, Rook does't use the trend of using block quotes for comments. To keep with prior form (and because there are leading `*`  anyway, let's just use `//`. 

At the same time, let's keep the line length to 100 chars max where possible, especially for comment blocks.

Reply:
sure will change accordingly